website/docs/r/apigatewayv2_stage.html.markdown

```diff
@@ -76,7 +76,7 @@ In addition to all arguments above, the following attributes are exported:
 or in an [`aws_iam_policy`](/docs/providers/aws/r/iam_policy.html) to authorize access to the [`@connections` API](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-how-to-call-websocket-api-connections.html).
 See the [Amazon API Gateway Developer Guide](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-websocket-control-access-iam.html) for details.
 * `invoke_url` - The URL to invoke the API pointing to the stage,
-  e.g. `wss://z4675bid1j.execute-api.eu-west-2.amazonaws.com/example-stage`
+  e.g. `wss://z4675bid1j.execute-api.eu-west-2.amazonaws.com/example-stage`, or `https://z4675bid1j.execute-api.eu-west-2.amazonaws.com/`
```

Comment:
I would also just note in the `execution_arn` bullet point item that the value is only set/computed for WebSocket APIs (as it was previously populated for both types)

Reply:
Done.